### PR TITLE
FS-4001: Add Assigned to you section to assessment overview

### PR DESCRIPTION
**Pull request by suegarner**
_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: app/blueprints/assessments/routes.py [hunk 1]
```diff
@@ -514,6 +514,8 @@ def assessor_type(fund_short_name: str, round_short_name: str):
     if not (selected_assessments := request.form.getlist("selected_assessments")):
         abort(500, "Required selected_assessments field to be populated")
 
+    from_form = request.form.get("from_form")
```
Comment (sfount) [inline app/blueprints/assessments/routes.py:517]: I wonder if something more explicit like `redirect_to_application_detail` would make the control flow read easier. I think originally `is_application` or `is_application_dashboard` which also sound like good options. 

Another option might be `action=application_overview` or an enum which would be more flexible but I suspect these are the only two that will be needed so thats likely overdoing it!

File: app/blueprints/assessments/routes.py [hunk 2]
```diff
@@ -1629,6 +1646,54 @@ def application(application_id):
             )
         )
 
+    thread_executor = ContextAwareExecutor(
```
Comment (sfount) [inline app/blueprints/assessments/routes.py:1649]: I'm not sure the savings from the thread executor mechanism are worth the complexity or if this logic shouldn't be encapsulated lower down. I know this logic was added before this PR so all good for it to go through.

I wonder if using the `async` functionality added to flask recently (relatively!) would be a less hands on way of managing waiting for multiple calls.

**Review by sfount — COMMENTED** [review]
Code and template are looking good! As discussed last week we thought we could cover this by adding an assertion for the default non-assigned behaviour on the existing application detail page tests first and then extend the mocks to add the scenarios where a non-lead and then lead assessor are assigned.

File: app/blueprints/assessments/routes.py (hunk 1 above)
Comment (suegarner) [reply app/blueprints/assessments/routes.py:517]: Renamed to redirect_to_application_detail

**Comment by sfount** [comment]
Ticket is currently blocked on design changes & access to environments - closing for combining 
 the frontends and we can support in gettin the branch pushed and PR raised against the new repo

**Pull request closed without merging**